### DISCUSSION TITLE
filter string column that has multiple value

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3483] [Q&A] [answered]
![image](https://github.com/pocketbase/pocketbase/assets/94731739/bd9b8976-ed6f-4576-8c2d-c2e79e69ab6b)
i want to filter all values that has fast in it

**Comment by ganigeorgiev** [comment, marked as answer]
You can try with the [`:each` modifier](https://pocketbase.io/docs/api-rules-and-filters/#each-modifier) and the `?=` operator (any/at-least-one):

```js
type:each ?= "fast"
```

Alternatively and depending on your use case, using the like/contains operator also could be used for loose comparison (_I expect the above direct match to be more performant than this_):

```js
type ~ '"fast"'
```
